src/containers/FactoryLoader.tsx

```diff
@@ -184,12 +197,13 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
       this.updateOverrideParams(key, val);
       factoryUrl.searchParams.append(key, val);
     });
-
     attrs.stackName = factoryUrl.toString();
+    const createPolicy = attrs['policies.create'] as CreatePolicy || 'perclick';
```

Comment:
I would move 'perclick' to a constant with default value, not sure if it's the way to go in typescript ;)

Reply:
If I understood correctly, it's not necessary because the `createPolicy` constant is treated as of `CreatePolicy` type and allows only defined values to be set.

Reply:
Ok. I have fixed it.